scrooge-generator/src/main/scala/com/twitter/mustache/ScalaObjectHandler.scala

```diff
@@ -0,0 +1,69 @@
+package com.twitter.mustache
```

Comment:
I'm worried that this will cause big problems for folks who depend both on scala-extensions and scrooge.

Can you test that it doesn't cause problems?  Also, we shouldn't expose this publicly–although scrooge-generator isn't technically a library, I'm worried that folks will use it anyway and it'll be painful to remove it after we move to java 8.